poms/bom/pom.xml

```diff
@@ -6055,18 +6048,6 @@
                 <artifactId>json-smart</artifactId>
                 <version>${json-smart.version}</version>
             </dependency>
-            <!-- TODO: Remove this https://github.com/apache/camel-quarkus/issues/3366 -->
-            <dependency>
-                <groupId>software.amazon.awssdk</groupId>
-                <artifactId>apache-client</artifactId>
-                <version>${awssdk.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>commons-logging</groupId>
-                        <artifactId>commons-logging</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
```

Comment:
I think this is the only part we should remove now that we upgrade to Quarkus 2.6. I think `software.amazon.awssdk:bom` in version synced with quarkiverse amazonservices is still needed to keep the versions of software.amazon.awssdk dependencies consistent.

Reply:
Ok, my note in #3366 is misleading then. Sorry about that.

Reply:
ok, let me change this.

Reply:
should be fine by now